feeds/wlan-ap/interAPcomm/Makefile

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libinterapcomm
+PKG_RELEASE:=1.0.0
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libinterapcomm
+  SECTION:=base
+  CATEGORY:=Base system
+  TITLE:=Inter AP communication library
+endef
+
+define Package/libinterapcomm/description
+ Inter AP communication library.
+endef
+
+define Build/Prepare
```

Comment:
you can drop the Build/Prepare section as it duplicates the functionality that is default behaviour

Reply:
It appears that the comments were not addressed in the latest version of the patch